index.html

```diff
@@ -37,6 +37,148 @@ <h1>Hey there. I'm <br />Subrat </h1>
         <i class="fa fa-chevron-down"></i>
       </span>
     </section>
+    <section id="Works">
+      <h1 class="sectionHeading">My Recent <br />Works </h1>
+      <div class="bar"></div>
+      <div class="card">
+        <div class="cardImage">
 
+        </div>

```

Comment:
I truly love your design and thorough implementation of the design template you choose. However, there are a few areas to improve upon;

- [ ] I would like to suggest that you put an **_image_** in the image placeholder instead of having a grey background as the image. This would make the portfolio look real and you did be fulfilling a project requirement for this milestone. 🙏

Reply:
There is no image on the provided template. Can I use any image of my choice ?